### ISSUE TITLE
Daedalus unacceptable bug

### ISSUE THREAD

**Issue by HuguesD** [input-output-hk/daedalus#3114]
Hi,

Using Daedalus version 5.2.0
Today, after a 1 ada transaction to myself, to empty the rewards, Daedalus displayed an erroneous ADA balance, adding twice the stake rewards to the addr1 balance.
There was no way to clear this error, except b y quitting Daedalus and starting it up again.

In parallel, I checked my balance on other client wallets, suc as Ledger live, which displayed the correct balance.

So I guess what is wrong is that Daedalus does not wait the end of the transaction and/or the correct sync with the blockchain to display the account balance.

This needs to be adressed as a priority as it destroys any credibility in Daedalus being a serious and stable wallet client.

**Comment by merlinana** [comment]
@HuguesD Talk to Ledger Support on the [Ledger Live support](https://explore-ledger.online/) so that your issue can be looked into.

Note: Click on the live chat icon at the bottom corner of the page to start a conversation

**Comment by HuguesD** [comment]
> @HuguesD Talk to Ledger Support on the [Ledger Live support](https://explore-ledger.online/) so that your issue can be looked into.
> 
> Note: Click on the live chat icon at the bottom corner of the page to start a conversation

The  mentioned bug is a daedalus bug and has nothing to do whatsoever with Ledger....

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
